project/plugins.sbt

```diff
@@ -10,12 +10,12 @@ resolvers ++= Seq(
   "Spy" at "https://files.couchbase.com/maven2/"
 )
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.11")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.19")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.0.3")
 
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.12")
 
-
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
 
+libraryDependencies += "org.vafer" % "jdeb" % "1.3" artifacts (Artifact("jdeb", "jar", "jar"))
```

Comment:
how were we building a deb package before? or is this a new thing?

Reply:
It's new as of sbt-native-packager 1.1.0, and a bit weird – in [this change to sbt-native-packager](https://github.com/sbt/sbt-native-packager/releases/tag/v1.1.0), the sbt plugin marks jdeb as a `provided` dependency, which IIUC means sbt assumes its available on the classpath. So we must provide it explicitly.